configFunctions.f90

```diff
@@ -91,21 +101,24 @@ subroutine findReactionsWithProductOrReactant( r, chs, arrayLen )
     return
   end subroutine findReactionsWithProductOrReactant
 
+  ! ----------------------------------------------------------------- !
```

Comment:
Ok. Ultimately, it's your code so what works for you is best! Do you always have a 72-character window in emacs, or do you use something wider? If so, I'd make it as wide as you use - modern displays are not limited to 72 chars before they become unreadable. If the visual aids work for you, then I won't push to remove them.